libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/ActivityLogExtensions.cs

```diff
@@ -47,7 +47,7 @@ public static RouteHandlerBuilder UseActivityLog(this IEndpointRouteBuilder buil
                             }).Take(100),
                     HasMore = activityLogEntries.Count > 100
                 };
-            }).RequireAuthorization();
+            }).WithTags("Activity log").RequireAuthorization();
```

Comment:
What does this do in practise?

Reply:
Adds the 'Activity log' tag to this specific endpoint so Swagger interface will show group 'Activity log' instead of 'main'.